src/models/m_rsc_update.erl

```diff
@@ -679,7 +679,7 @@ props_filter([{page_path, Path}|T], Acc, Context) ->
                 Empty when Empty == undefined; Empty == []; Empty == <<>> ->
                     props_filter(T, [{page_path, undefined} | Acc], Context);
                 _ ->
-                    P = [ $/ | string:strip(z_utils:url_path_encode(Path), both, $/) ],
+                    P = [ $/ | string:strip(z_url:url_path_encode(Path), both, $/) ],
```

Comment:
Line 682 is too long:                     P = [ $/ | string:strip(z_url:url_path_encode(Path), both, $/) ],.